src/components/common/animations/ClaimButton/ClaimButton.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Lottie from 'lottie-react-native'
 import { Platform, TouchableOpacity } from 'react-native'
-import _set from 'lodash/set'
+import { set as _set } from 'lodash'
```

Comment:
please use aliases in the case of collision only

here just:
```
import { set } from 'lodash'

set(animationData, 'layers[4].ks.p.k[0].s[0]', this.getGap())
set(animationData, 'layers[4].ks.p.k[1].s[0]', this.getGap())
```

also, it would be good to calculate gap once then just reuse constant:

``` 
const gap = this.getGap()

set(animationData, 'layers[4].ks.p.k[0].s[0]', gap)
// ...
```

Reply:
done